lib/std/math/cbrt.zig

```diff
@@ -87,9 +87,9 @@ fn cbrt64(x: f64) f64 {
         u = @as(u64, @bitCast(x * 0x1.0p54));
         hx = @as(u32, @intCast(u >> 32)) & 0x7FFFFFFF;
 
-        // cbrt(0) is itself
+        // cbrt(+-0) = itself
```

Comment:
Might make sense to keep the original `cbrt(0) is itself` marker from musl, seems like a nice guidance marker

Reply:
The comment in the [cbrtf musl function](https://git.musl-libc.org/cgit/musl/tree/src/math/cbrtf.c) is `cbrt(+-0) is itself`. Since that is what was put in the `cbrt32` function and not here, I figured it might be part of what caused the bug in the first place. I could change it back, but would you also want the `cbrt32` comment to change accordingly?

Reply:
Ah, looked at the wrong one - nevermind!